docs/source/checks/simple.rst

```diff
@@ -0,0 +1,175 @@
+.. _simple:
+
+Simple Check
+====================
```

Comment:
```suggestion
======
```

Reply:
not really that important I would say, but okay